dexs/idex-v4/index.ts

```diff
@@ -0,0 +1,26 @@
+import type { SimpleAdapter } from "../../adapters/types";
+import { httpGet } from "../../utils/fetchURL";
+
+const URL = "https://api.idex.io/v4/exchange";
+
+const fetch = async () => {
+  const response = await httpGet(URL);
+  const { totalVolume, volume24h } = response;
+
+  return {
+    totalVolume: totalVolume?.toString(),
+    dailyVolume: volume24h?.toString(),
+  };
+};
+
+const adapter: SimpleAdapter = {
+  adapter: {
+    "idex-v4": {
```

Comment:
chain name should `idex`  ?

Reply:
I misunderstood this key - the chain name is XCHAIN, the exchange name running there is IDEX

Reply:
xchain will likely get confused with the multiple other xchains there are, such as avalanche's, imo better to go with idex

Reply:
Updated